src/services/azblob/backend.rs

```diff
@@ -124,6 +128,18 @@ impl Builder {
         self
     }
 
+    /// Set sas_token of this backend.
+    ///
+    /// - If sas_token is set, we will take user's input first.
+    /// - If not, we will try to load it from environment.
```

Comment:
Please add a link to https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview